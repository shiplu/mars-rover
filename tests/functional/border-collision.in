5 5
1 2 N
LML
3 3 E
MMR
