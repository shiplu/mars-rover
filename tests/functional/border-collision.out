1 2 W STOPPED
5 3 S OPERATIONAL
